apps/bfd-pipeline/bfd-pipeline-rda-bridge/pom.xml

```diff
@@ -59,6 +59,12 @@
             <artifactId>lombok</artifactId>
             <scope>provided</scope>
         </dependency>
+        <!-- Templating Engine -->
+        <dependency>
+            <groupId>org.freemarker</groupId>
+            <artifactId>freemarker</artifactId>
+            <version>2.3.31</version>
+        </dependency>
```

Comment:
Is this a new dependency for bfd? (Also version should be hoisted up to the properties)

Reply:
It is, this is also not a deployed application in any sense, it's only run on a local development machine, but it is in fact a new dependency.

Reply:
we should check that box in the PR description for new software dependency then and get approval from @StewGoin

Reply:
@StewGoin The RDABridge application is something the PACA devs run on their laptops to convert Synthea RIF data sets into a set of JSON files that match the RDA API specification.  We do this after the BFD publishes a new Synthea release to get the partially adjudicated data in the SBX environment in alignment with the adjudicated data.

In this PR, we added a new lib (freemaker) to help us with templating needs.  Specifically, when we run the bridge locally, we optionally have the synthetic MBIs used in the conversion outputted to a file so we can use them for attribution in the BCDA SBX database.

Please let us know if you have any objections and/or question to us using the freemaker lib.

Reply:
No objections, I appreciate the transparency in it since it's part of the ecosystem supporting PACA even if its not running in the "application" environment itself.